docs/quickstart-aks.md

```diff
@@ -0,0 +1,336 @@
+    ```console
+    echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheezy main" | \
+         sudo tee /etc/apt/sources.list.d/azure-cli.list
+    ```
+1. Run the following commands to install the Azure CLI and its dependencies:
+    ```console
+    sudo apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
+    sudo apt-get install apt-transport-https
+    sudo apt-get update && sudo apt-get install azure-cli
+    ```
+
+### Install the Kubernetes CLI
+
+Install `kubectl` by running the following command:
+
+```console
+az acs kubernetes install-cli
+```
+
+### Install the Helm CLI
+
+[Helm](https://github.com/kubernetes/helm) is a tool for installing pre-configured applications on Kubernetes.
+Install `helm` by running the following command:
+
+#### MacOS
+
+```console
+brew install kubernetes-helm
+```
+
+#### Windows**
+
+1. Download the latest [Helm release](https://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-windows-amd64.tar.gz).
+1. Decompress the tar file.
+1. Copy **helm.exe** to a directory on your PATH.
+
+#### Linux
+
+```console
+curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
+```
+
+---
+
+## Cluster Setup
+
+Now that we have all the tools, we need a Kubernetes cluster with Open Service Broker for Azure configured.
+
+### Configure your Azure account
+
+First let's identify your Azure subscription and save it for use later on in the quickstart.
+
+1. Run `az login` and follow the instructions in the command output to authorize `az` to use your account
+1. List your Azure subscriptions:
+    ```console
+    az account list -o table
+    ```
+1. Copy your subscription ID and save it in an environment variable:
+
+    **Bash**
+    ```console
+    export AZURE_SUBSCRIPTION_ID="<SubscriptionId>"
+    ```
+
+    **PowerShell**
+    ```console
+    > $env:AZURE_SUBSCRIPTION_ID = "<SubscriptionId>"
+    ```
+
+### Create a service principal
+
+This creates an identity for Open Service Broker for Azure to use when provisioning
+resources on your account on behalf of Kubernetes.
+
+1. Create a service principal with RBAC enabled for the quickstart:
+    ```console
+    az ad sp create-for-rbac --name osba-quickstart -o table
+    ```
+1. Save the values from the command output in environment variables:
+
+    **Bash**
+    ```console
+    export AZURE_TENANT_ID=<Tenant>
+    export AZURE_CLIENT_ID=<AppId>
+    export AZURE_CLIENT_SECRET=<Password>
+    ```
+
+    **PowerShell**
+    ```console
+    > $env:AZURE_TENANT_ID = "<Tenant>"
+    > $env:AZURE_CLIENT_ID = "<AppId>"
+    > $env:AZURE_CLIENT_SECRET = "<Password>"
+    ```
+
+### Create a Kubernetes cluster using AKS
+
+Next we will create a managed Kubernetes cluster using AKS. AKS will create a managed Kubernetes cluster for you. Once the cluster is created, geting started with OSBA is very similar to doing so on Minikube, with a few exceptions: 
```

Comment:
Let's link to the minikube quickstart here since I believe we are referencing it.

Reply:
Good idea!